solidity/contracts/liquidity-protection/interfaces/ITransferPositionEventCallback.sol

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: SEE LICENSE IN LICENSE
```

Comment:
`ITransferPositionEventClient`?
`ITransferPositionEventListener`?
`ITransferPositionEventReceiver`?
Surely there's a standard design pattern name for that.

Reply:
`ITransferPositionEventReceiver`, perhaps? @yudilevi WDYT?

Reply:
I've talked with @yudilevi and we've agreed `ITransferPositionCallback` 😅